### PR TITLE
Make ipc:// sockets work.

### PR DESCRIPTION
**Pull request by FGasper**
Issue #2

### CODE REVIEW

**Comment by jacquesg** [comment]
Looks good, I think you just need to update ```t/02-socket.t```

**Comment by FGasper** [comment]
@jacquesg I already updated it .. ?

**Comment by jacquesg** [comment]
The test failure on Azure/Appveyor is due to the 2 extra lines that were added. One of the tests looks for that line in some output.

File: t/02-socket.t [hunk 1]
```diff
@@ -3,6 +3,8 @@
 use Test::More;
 use ZMQ::Raw;
 
+use File::Temp;
```
Comment (jacquesg) [inline t/02-socket.t]: This introduces 2 extra lines.

Comment (jacquesg) [reply t/02-socket.t]: Line 94 later on needs to be adjusted.

**Review by jacquesg — CHANGES_REQUESTED** [review]

File: t/02-socket.t (hunk 1 above)
Comment (FGasper) [reply t/02-socket.t]: Ah ok, I’ll look.

**Review by jacquesg — APPROVED** [review]

**Comment by jacquesg** [comment]
Cherry-pick onto master, thank you.

**Pull request closed without merging**